installers/MacInstaller/CMakeDMGSetup.scpt

```diff
@@ -41,8 +41,6 @@ on run argv
         set the bounds to { 400, 000, 900, 710 }
```

Comment:
Could we adjust the window size to something like: `set the bounds to { 400, 000, 900, 540 }`